bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/creators/ValueCreator.java

```diff
@@ -830,10 +829,45 @@ public static BMap<BString, Object> createRecordValue(Module packageId, String r
     public static BMap<BString, Object> createReadonlyRecordValue(Module packageId, String recordTypeName,
```

Comment:
Shall we deprecate other createReadonlyRecordValue takes Map as a parameter.

Reply:
Discussed not to deprecate these.